modules/Microsoft.DevTestLab/labs/.test/common/deploy.test.bicep

```
@@ -0,0 +1,245 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.devtestlab.labs-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dtlcom'
```

```suggestion
param serviceShort string = 'dtllcom'
```
I guess 😄 Beware, requires regeneration of the readme's

---

Fixed